aiida/manage/configuration/profile.py

```diff
@@ -130,12 +131,18 @@ def __init__(self, name, attributes, from_config=False):
 
     def get_repository(self) -> 'Repository':
         """Return the repository configured for this profile."""
-        from disk_objectstore import Container
         from aiida.repository import Repository
+
+        backend = self.get_backend()
```

Comment:
Looking at failed CI, found here is a mistake. In case of disturbing you with too many push to your branch, I will fix this later with more valid commits.

```suggestion
        backend = self.get_repository_backend()
```